cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/FlowletTwillRunnable.java

```diff
@@ -51,9 +47,4 @@ public void handleCommand(Command command) throws Exception {
       LOG.debug("Command failure ignored", e);
     }
   }
-
-  @Override
-  protected Arguments resolveScope(Arguments arguments) {
```

Comment:
Do you need to get rid of this? Couldn't see how this is used besides forcing you to extract scope in the FlowletProgramRunner. It's simpler if a flowlet only knows its own arguments and not those of other flowlets.